### PR TITLE
#851 local Netbout setup runs without requiring tests to be run beforehand

### PR DESCRIPTION
**Pull request by original-brownbear**
#851 is resolved by this.

PR looks bigger than it is:
* Created new profile `casperjs` that does nto run when tests are skipped
* Moved all casperjs related things ( install and test run ) in there
* Made the requirements for local run via the `hit-refresh` profile load per default
 * I think this is ok now, given that we only load the things for local run, not the casperjs install since I moved it out of the takes-test block
* Renamed profile `takes-test` to `takes-setup` since this does not invoke the casperjs tests like it did before anymore now

### CODE REVIEW

**Comment by karato** [comment]
@original-brownbear Many thanks, I will find someone to review it before we merge

**Comment by karato** [comment]
@mkordas do this review

**Comment by mkordas** [comment]
@original-brownbear sorry, not time for it today, I'll treat it as priority in the morning

File: netbout-web/pom.xml [hunk 1]
```diff
@@ -552,19 +574,26 @@
                                     <goal>exec</goal>
                                 </goals>
                                 <configuration>
-                                    <executable>${casper.executable}</executable>
-                                    <workingDirectory>${basedir}</workingDirectory>
+                                    <executable>${casper.executable}
```
Comment (mkordas) [inline netbout-web/pom.xml]: @original-brownbear `${casper.executable}` should be on its own line

File: netbout-web/pom.xml [hunk 2]
```diff
@@ -529,11 +548,14 @@
                                 </goals>
                                 <configuration>
                                     <executable>git</executable>
-                                    <workingDirectory>${project.build.directory}</workingDirectory>
+                                    <workingDirectory>${project.build.directory}
+                                    </workingDirectory>
```
Comment (mkordas) [inline netbout-web/pom.xml]: @original-brownbear same here, please check formatting

File: netbout-web/pom.xml [hunk 3]
```diff
@@ -552,19 +574,26 @@
                                     <goal>exec</goal>
                                 </goals>
                                 <configuration>
-                                    <executable>${casper.executable}</executable>
-                                    <workingDirectory>${basedir}</workingDirectory>
+                                    <executable>${casper.executable}
+                                    </executable>
+                                    <workingDirectory>${basedir}
+                                    </workingDirectory>
```
Comment (mkordas) [inline netbout-web/pom.xml]: @original-brownbear and here too, please join lines or put it into 3 lines

File: netbout-web/pom.xml [hunk 4]
```diff
@@ -552,19 +574,26 @@
...
                                         <argument>--no-colors</argument>
                                         <argument>--concise</argument>
-                                        <argument>--home=http://localhost:${takes.port}</argument>
+                                        <argument>
+                                            --home=http://localhost:${takes.port}
+                                        </argument>
                                         <argument>--engine=phantomjs</argument>
-                                        <argument>${basedir}/src/test/casperjs</argument>
+                                        <argument>${basedir}/src/test/casperjs
+                                        </argument>
```
Comment (mkordas) [inline netbout-web/pom.xml]: @original-brownbear please revert all unnecessary changes

**Comment by mkordas** [comment]
@original-brownbear just style comments for now

**Comment by original-brownbear** [comment]
@mkordas fixed those for now then :)

**Comment by mkordas** [comment]
@original-brownbear fixes look good

**Comment by mkordas** [comment]
@original-brownbear can we also document `-DskipTests` in https://github.com/yegor256/netbout#run-locally-and-test-it? Testing is not always necessary before running

**Comment by original-brownbear** [comment]
@mkordas done, check it out :)

File: README.md [hunk 5]
```diff
@@ -81,6 +81,13 @@ $ mvn clean install -Phit-refresh -Dport=8080
 
 In a minute the site is ready at `http://localhost:8080`
 
+In case you are looking for an even quicker startup or experience failing tests,
+but still want to run the site, you can skip the tests by running:
+
+```
+$ mvn clean install -DskipTests=true -Phit-refresh -Dport=8080
```
Comment (mkordas) [inline README.md]: @original-brownbear I think just `-DskipTests` is enough, no need for `=true`

**Comment by mkordas** [comment]
@original-brownbear nice, just one very minor thing

**Comment by original-brownbear** [comment]
@mkordas fixed :)

**Comment by mkordas** [comment]
@original-brownbear good work

**Comment by mkordas** [comment]
@rultor merge

**Comment by dmzaytsev** [comment]
@original-brownbear perfect
thanks

**Comment by dmzaytsev** [comment]
@rultor merge

**Comment by karato** [comment]
@elenavolokhova please, let us know what do you think about this ticket, according to our [QA rules](http://at.teamed.io/qa.html)

**Comment by karato** [comment]
@rultor deploy

**Comment by elenavolokhova** [comment]
@karato Looks good!

**Comment by karato** [comment]
> @karato Looks good!

@elenavolokhova thanks a lot ;)

**Comment by karato** [comment]
@mkordas since quality is good, I just added 10 mins to @dmzaytsev (our architect) in transaction 79354805

**Comment by karato** [comment]
@mkordas just added 10 mins to @elenavolokhova (for QA), payment ID is `79371620`. thanks, paid, **27 mins** to your account, payment ID is `56dc054f84d289032d0003a7`, task took 17 hours and 59 mins to complete. extra minutes for review comments (c=12). +27 added to your rating, at the moment it is: [+8187](http://www.netbout.com/b/36080?open=rating)
